WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -32,9 +32,13 @@ Most of the features introduced in the Semi-Annual Channel will be rolled up int
 The Semi-Annual Channel is available to volume-licensed customers with [Software Assurance](https://www.microsoft.com/licensing/licensing-programs/software-assurance-default.aspx), as well as via the Azure Marketplace or other cloud/hosting service providers and loyalty programs such as Visual Studio Subscriptions.
 
 > [!Note]
-> **The current Semi-Annual Channel release is Windows Server, version 2004**. If you want to put servers in this channel, you should install Windows Server, version 2004, which can be installed in Server Core mode or as Nano Server run in a container. In-place upgrades from a long-term servicing channel release aren't supported because they are in **different release channels**. Semi-Annual Channel releases aren't updates – it's the next Windows Server release in the Semi-Annual Channel.
+> **The current Semi-Annual Channel release is Windows Server, version 2004**. If you want to put servers in this channel, you should install Windows Server, version 2004, which can be installed in Server Core mode or as Nano Server run in a container. In-place upgrades from a long-term servicing channel release aren't supported because they are in **different release channels**. This applies vice versa. You cannot upgrade or change from Semi-Annual Channel to a LTSC channel, without a clean installation.
```

Comment:
```suggestion
> **The current Semi-Annual Channel release is Windows Server, version 2004**. If you want to put servers in this channel, you should install Windows Server, version 2004, which can be installed in Server Core mode or as Nano Server run in a container. In-place upgrades from a long-term servicing channel release aren't supported because they are in **different release channels**. This applies vice versa. You cannot upgrade or change from Semi-Annual Channel to an LTSC channel without a clean installation.
```
Codestyle: initialisms and acronyms should be preceded by the articles "a" or "an" depending on how the initials are pronounced (said out loud), not how they are written. In this case, "LTSC" is pronounced "Ell-Tee-Ess-Cee" and therefore requires it to be preceded as if it was a word starting with a pronounced vowel. Comma removal: the final clause does not work as a stand-alone clause, but would be if a verb was present.